app/assets/javascripts/discourse/tests/acceptance/user-preferences-tracking-test.js

```diff
@@ -0,0 +1,220 @@
+import { acceptance, exists } from "discourse/tests/helpers/qunit-helpers";
+import { click, visit } from "@ember/test-helpers";
+import { test } from "qunit";
+import selectKit from "discourse/tests/helpers/select-kit-helper";
+
+acceptance("User Preferences - Tracking", function (needs) {
```

Comment:
Great tests, very nice. The middle two could be candidates for system specs as well, whereas the top and bottom ones are probably too granular.